storage/country_tree.hpp

```diff
@@ -3,9 +3,12 @@
 #include "base/assert.hpp"
 
 #include "std/algorithm.hpp"
-#include "std/unique_ptr.hpp"
+#include "std/shared_ptr.hpp"
 #include "std/vector.hpp"
 
+template <class T>
+bool IsEqual(T const & v1, T const & v2) { return !(v1 < v2) && !(v2 < v1); }
```

Comment:
This method is used only in CountryTreeFacade class.
I'm moving it to CountryTreeFacade class and implementing it as a private static method:
```
private:
  static bool IsEqual(T const & v1, T const & v2) { return !(v1 < v2) && !(v2 < v1); }
```